specification/nexusidentity/resource-manager/Microsoft.NexusIdentity/readme.md

```diff
@@ -0,0 +1,43 @@
+# Nexus Identity
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for Nexus Identity.
+
+## Getting Started
+
+To build the SDKs for My API, simply install AutoRest via `npm` (`npm install -g autorest`) and then run:
+
+> `autorest readme.md`
+
+To see additional help and options, run:
+
+> `autorest --help`
+
+For other options on installation see [Installing AutoRest](https://aka.ms/autorest/install) on the AutoRest github page.
+
+---
+
+## Configuration
+
+### Basic Information
+
+These are the global settings for the Nexus Identity.
+
+```yaml
+openapi-type: arm
+tag: package-2023-06-15
+```
+
+### Tag: package-2023-06-15
+
+These settings apply only when `--tag=package-2023-06-15` is specified on the command line.
+
+```yaml $(tag) == 'package-2023-06-15'
+input-file:
+  - stable/2023-06-15/identityset.json
+  - stable/2023-06-15/identityController.json
+suppressions:
+  - code: ProvisioningStateMustBeReadOnly
```

Comment:
I believe the recommended fix is to set `use-read-only-status-schema` in `tspconfig.yaml`, rather than suppress this rule.

https://github.com/Azure/typespec-azure/blob/d41600db0e74b6bd682a7d7a033ce4a83f77ba5b/eng/feeds/arm/tspconfig.yaml#L5
https://azure.github.io/typespec-azure/docs/emitters/typespec-autorest/reference/emitter#use-read-only-status-schema

Reply:
CC: @markcowl

Reply:
changed it